app/misc.py

```diff
@@ -229,6 +237,14 @@ def get_global_stylesheet(self):
         return ''
 
 
+def is_user_admin(uid):
+    get_admin_users = User.select(User.name).join(UserMetadata).where((UserMetadata.key == 'admin') & (UserMetadata.value == '1'))
```

Comment:
I think you want to select `User.uid` here.

Reply:
Yup, that helps!